### ISSUE TITLE
[BUG] Does not work (at all) when used with Foundry 11 and Shadowdark game system

### ISSUE THREAD

**Issue by tbrockwood** [CDeenen/NotYourTurn#33]
Foundry version 11.315
Shadowdark 2.2.2.

Modules installed: I've disabled everything except NotYourTurn, but it still doesn't work. 

When I put the PCs in initiative order, roll init, and then start the combat, NotYourTurn has no effect. Players can move around as they wish at any time, even when out of their turn order. It's as if the module's not active at all. I've tried all the settings, Dialog shows no dialog when they act, autoblock has no effect, etc.

**Comment by STBaf** [comment]
Must mention never tried it with Shadowdark (or even played Shadowdark). I need to install that to my testing environment and see if it works.
While I'm doing this have you tried to install/use another game system (let's say the probably most popular: DnD5e) and tried if that works on your side ? if yes that could be a indicator that the Shadowdark game system making modifications how the core combat tracker is working which will maybe get the Not Your Turn! module in trouble.

**Comment by tbrockwood** [comment]
I just tried it in Basic Fantasy RPG (I don't use 5e, so I'll have to try that later), and instead of autoblocking movement, it gives the "It's not your turn" warning, but still lets them move. I checked the settings, and they were all set to autoblock

**Comment by tbrockwood** [comment]
Just tried it in 5e on v11. Still doesn't autoblock. The player can move, but they get a popup that says "It's not your turn" even though it's set to autoblock

**Comment by STBaf** [comment]
Tested with Shadowdark 3.0.0 and FoundryVTT 3.15, only Not Your Turn! Module Active, GM + Player Account logged in separte with following NYT! basic configuration:

![grafik](https://github.com/user-attachments/assets/6c7bc4b7-49d7-4330-8e3d-6cf0be9226e8)

Working as expected: Player could not move with message that is is blocked (=autoblock behavior), GM was able to move with warning (=Warning only). Sure you have no other module active ?

**Comment by STBaf** [comment]
Adding: Installed 2.2.2 - no problem, too. Can't reproduce

**Comment by tbrockwood** [comment]
I have no other modules installed, and updated 5e to the lastest version and made sure NYT! was current. I created two dummy players and logged in via an incognito window as one, then set up a combat on the GM's browser. Any time NYT should block movement, instead (on the player's screen) it gives a little notification of "It's not your turn" but doesn't block me. I've made sure to follow your settings from above to be sure. 

If I set it to do a GM request move, that triggers normally. It's just autoblock that's not working. Is it possible that it's because I'm on foundry version 11.315 and not 12?

**Comment by STBaf** [comment]
> Is it possible that it's because I'm on foundry version 11.315 and not 12?

No, I'm running my games on v11.315 too at the moment and using Not Your Turn! in my own games all the time ;) Is your game up 24/7 ? any chance I can take a look at it directly since I can not reproduce here (so I'm not able to analyze or fix anything). Or can you DM me on discord when you can put up your game so I can take a look ?

**Comment by STBaf** [comment]
Could not be reproduced, no fix done
